setup.cfg

```diff
@@ -1,14 +1,16 @@
 [metadata]
 name = matrix_common
-description = "Common utilities for Synapse, Sydent and Sygnal"
+description = Common utilities for Synapse, Sydent and Sygnal
 long_description = file: README.md
 long_description_content_type = text/markdown
-version = 0.0.0
+version = 1.1.0
 
 classifiers =
   License :: OSI Approved :: Apache Software License
 
 [options]
+packages =
+  matrix_common
```

Comment:
Is this the right way to _not_ publish an empty package?

Reply:
I'm not really sure what you mean by this? I think usually you can either specify the packages manually, as you did, or use `find:` (and potentially exclude any tests, etc.)

Reply:
I was uncertain at the time because I was expecting `synapse-module-cookiecutter-template` to work out of the box.
I've PRed an equivalent change into the template so it's all okay now.